lib/temporal/configuration.rb

```diff
@@ -26,12 +26,13 @@ class Configuration
 
     def initialize
       @client_type = :grpc
-      @logger = Temporal::Logger.new(STDOUT, progname: 'temporal_client')
+      @logger = Logger.new(STDOUT, progname: 'temporal_client')
```

Comment:
What is the reason for this change?

Reply:
This is a mistake, it is legacy from the code change existing before Temporal::Logger. 
I will remove it.